src/subscription.js

```diff
@@ -185,6 +188,13 @@ function Subscription(pubsub, name, options) {
     options.flowControl
   );
 
+  this.batching = extend(
+    {
+      maxMilliseconds: 100,
```

Comment:
@kir-titievsky does the PubSub team have any recommendations and/or opinions on how long the client should buffer acks for?